lib/blockstore/index.js

```diff
@@ -0,0 +1,42 @@
+/*!
+ * blockstore/index.js - bitcoin blockstore for bcoin
+ * Copyright (c) 2019, Braydon Fuller (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+const {join} = require('path');
+
+const AbstractBlockStore = require('./abstract');
+const LevelBlockStore = require('./level');
+const FileBlockStore = require('./file');
+
+/**
+ * @module blockstore
+ */
+
+exports.create = (options) => {
+  const location = join(options.prefix, 'blocks');
+
+  if (options.memory) {
```

Comment:
I'm okay with using `LevelBlockStore` in the case of memory usage. This method will be necessary so that it can be replaced with other stores in various cases.